server/config/default.js

```diff
@@ -1,8 +1,11 @@
+// eslint-disable-next-line
```

Comment:
LOL does it npm run build and npm run start? If that works then fine by me

Reply:
we need this because the .env file cant be placed in server folder or else it wont resolve the path

Reply:
and ya it builds and runs

Reply:
this may break the deployment actually fuck, depends on where and how the env variables are resolved to

Reply:
doesn’t process.env fallback to the shell environment if an env file isn’t available?

Reply:
true, so deployment should be ok it was working earlier. This fixes local use